docs/management.asciidoc

```diff
@@ -41,5 +41,6 @@ include::management/managing-beats.asciidoc[]
 
 include::management/managing-remote-clusters.asciidoc[]
 
+include::management/snapshot-repositories/index.asciidoc[]
```

Comment:
the app will soon (7.3) be re-named to Snapshot and Restore. wondering if we should future-proof the URLs and directory names here? `snapshot-restore` instead of `snapshot-repositories`? this is only relevant if we care about not breaking doc URLs - if it's not that important, then I think leaving it is fine.